lib/masamune/transform/operator.rb

```diff
@@ -41,6 +42,14 @@ def target
       @presenters.key?(target_store.try(:type)) ? @presenters[target_store.try(:type)].new(@target) : @target
     end
 
+    def helper
+      (@helper || SimpleDelegator).new(self)
```

Comment:
a helper has access to all template `locals` and is itself a local variable during template evalutation